contrib/node/src/python/pants/contrib/node/BUILD

```diff
@@ -9,6 +9,7 @@ contrib_plugin(
     'contrib/node/src/python/pants/contrib/node/tasks',
     'src/python/pants/build_graph',
     'src/python/pants/goal:task_registrar',
+    'src/python/pants/backend/codegen/thrift/node',
```

Comment:
I think this dep is broken: there is no target at `src/python/pants/backend/codegen/thrift/node`. Your Task lives in ` contrib/node/src/python/pants/contrib/node/tasks`, so I think that you should just drop this line.

Reply:
Yep! That was from way back when we were putting node gen in with codegen instead of in node